### PR TITLE
Move cobalt/fonts to starboard/fonts

### PR DESCRIPTION
**Pull request by kaidokert**
Removes reverse dependencies from Starboard back to Cobalt. No targets within Starboard should refer to anything in /cobalt toplevel directory.

Test-On-Device: true

b/262794004

### CODE REVIEW

**Review by andrewsavage1 — APPROVED** [review]
This seems fine, but I would recommend testing heavily in both external/internal jobs and getting someone from evergreen to take a look if only to see if some other documentation should be updated.

**Comment by kaidokert** [comment]
> This seems fine, but I would recommend testing heavily in both external/internal jobs and getting someone from evergreen to take a look if only to see if some other documentation should be updated.

Ack yes. I'm trying to test as well as i can, internal jobs are a bit shaky at the moment.

File: starboard/build/config/BUILDCONFIG.gn [hunk 1]
```diff
@@ -359,11 +359,11 @@ template("evergreen_loader") {
     "//starboard/loader_app:copy_loader_app_content",
   ]
   if (cobalt_font_package == "empty") {
-    shared_data_deps += [ "//cobalt/content/fonts:copy_font_data" ]
+    shared_data_deps += [ "//staboard/content/fonts:copy_font_data" ]
```
Comment (oxve) [inline starboard/build/config/BUILDCONFIG.gn:362]: `staboard` -> `starboard`

**Review by oxve — APPROVED** [review]

File: starboard/build/config/BUILDCONFIG.gn (hunk 1 above)
Comment (kaidokert) [reply starboard/build/config/BUILDCONFIG.gn]: Oops, thanks for catching ! This would have tripped up at least a few partners

**Pull request merged**